pkgs/development/python-modules/pysparse/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, setuptools
+, blas
```

Comment:
Is there any reason this can't be parsed in as openblas to avoid having to pass in `{ blas = pkgs.openblas }` in the callPackage statement?

Reply:
This is so that in the future someone could provide mkl, atlas, blas instead. We do a similar thing with numpy

Reply:
fair enough